charts/eks-node-monitoring-agent/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: eks-node-monitoring-agent
-version: 1.2.0
-appVersion: 1.2.0
+version: 1.3.0
+appVersion: 1.3.0
```

no changes to the version of the agent image
```suggestion
appVersion: 1.2.0
```

---

find and replace failed me, sorry! 😅 

pushed the fix